book/07-error-handling.html

```diff
@@ -446,7 +446,7 @@ <h1>Declaring Exceptions Using <code>[@@deriving sexp]</code></h1>
           <h2>Helper Functions for Throwing Exceptions</h2>
 
           <p>
-	    OCaml and Core provide a number of helper functions to
+	    OCaml and Base provide a number of helper functions to
```

Comment:
Just a nit here, but is "OCaml and Base" necessary, as opposed to just "Base"?  Anything that OCaml provides is re-exported by Base, often with different conventions, so adding OCaml here seems redundant.

Reply:
Fixed.